config/sync/filter.format.os2loop_documents_table.yml

```diff
@@ -3,7 +3,7 @@ langcode: en
 status: true
 dependencies:
   module:
-    - media
+    - toc_filter
```

Comment:
I assume drupal figured out it no longer needs the media dependency in this config. It makes sense since there is nothing in the filter that references the media module, maybe the module has started cleaning up after itself when navigating the form with a recent update.

Unlike the media module both toc_filter and token_filter are probably not cleaning up when disabling the filter in the form ui.